components/cli/internal/test/kubecli.go

```diff
@@ -67,15 +66,9 @@ func WithServices(services map[string]kubernetes.Services) func(*MockKubeCli) {
 	}
 }
 
-func WithCellLogs(cellLogs map[string]string) func(*MockKubeCli) {
+func WithRunningInstance(isInstanceAvailable bool) func(*MockKubeCli) {
```

Comment:
Is this correct? Does this method cover the scenario you intend to mock? It seems that the requirement is to check whether a mock instance (cell/composite) is available. In that case, we can mock a cell instance using the functional option "WithCells" or mock composite instance with "WithCells" (not implemented yet).

Then the MockKubeCli should implement "GetInstanceAvailability" and return the instance availability. This way you don't have to hardcode the instance availability in unit tests and it mocks the real scenario.

Reply:
Now we don't need WithCells and WithComposites options since we implement a separate function to check the instance availability. (Previously,  the WithCells and WithComposites mocked some instance logs to indicate an instance is available)
Please correct me  if I'm wrong

Reply:
Previously "WithCellLogs", "WithComponentLogs" mocked instance logs and "WithCells" and "WithComposites" were not used in logs command unit tests if I am not mistaken and that indicated the availability of an instance, yes.
IMHO in unit tests, we should mock cells, composites, logs, virtual services, etc. and not instance availability("WithRunningInstance"), which should be derived using mocked instances.

Reply:
Yes. Removed `isInstanceAvailable` from kubecli